cobalt/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -52,8 +52,12 @@
 import java.util.HashMap;
 import java.util.Locale;
 import java.util.TimeZone;
+import org.chromium.base.annotations.CalledByNative;
+import org.chromium.base.annotations.JNINamespace;
+import org.chromium.base.annotations.NativeMethods;
 
 /** Implementation of the required JNI methods called by the Starboard C++ code. */
+@JNINamespace("starboard::android::shared")
```

Comment:
Looking at the use of JNINamespace in Chromium [1], I'd say this should be `@JNINamespace(cobalt)`, WDYT?



[1] https://source.chromium.org/search?q=@JNINamespace&ss=chromium

Reply:
This has to match the namespace in which the corresponding JNI native class is defined.